jsonlinter.py

```diff
@@ -0,0 +1,98 @@
+# python3 lintjson.py
+# William Rehwinkel / obiesource, 2022
+
+import sys
+import os
+import json
+import traceback
+
+def getJsonFiles():
+    # list all files recursively borrowed from https://pythonexamples.org/python-get-list-of-all-files-in-directory-and-sub-directories/
+    filelist = list()
+    path = "./"
+    
+    for root, dirs, files in os.walk(path):
+	    for file in files:
+            #append the file name to the list
+		    filelist.append(os.path.join(root,file))
+    
+    #print all the file names
+    jsonFiles = list()
+    
+    for name in filelist:
+        if name.lower().endswith(".json"):
+            jsonFiles.append(name)
+    return jsonFiles
+
+MEMBERS_REQUIRED = {
+        "name": "",
+        "pronouns": "",
+        "class": "",
```

Comment:
I assume the `"class"` field is to contain the current CS class level someone is in. I think since we have and welcome members who aren't in CS classes, and since at higher levels people take more than one cs class at a time, we should rename this to `"classes"`, and some people can put "None" and others can list the classes they're taking.

Reply:
`"class"` is the graduation year (for example, `"'24"`)